charts/gsp-cluster/values.yaml

```diff
@@ -251,7 +251,7 @@ secrets:
   private_key: ""
 
 letsencrypt:
-  email: daniel.blair@digital.cabinet-office.gov.uk
```

Comment:
I don't think this is used anywhere, is it? I clearly intended to template this out at some point and then never quite got there...